infra/integration-image/Dockerfile.test

```diff
@@ -0,0 +1,37 @@
+# docker build -t vic-test -f Dockerfile.test .
+FROM golang:1.6.2
+
+RUN apt-get update
+RUN apt-get install -y jq time
+RUN curl -sSL https://get.docker.com/ | sh
```

Comment:
This looks wrong as it returns the latest version, we should request a specific version (or use distro package)